packages/edit-site/src/components/routes/redirect-to-homepage.js

```diff
@@ -48,7 +55,7 @@ async function getHomepageParams( siteUrl ) {
 		.then( ( { data } ) => data );
```

Comment:
I can probably add some better error handling to these lines to improve the user's error message.

Reply:
`apiFetch` will throw if there's a 4xx or 5xx error, right? Would be good to access any error message that comes back from that, in case it's useful for debugging.

Reply:
The `window.fetch` won't throw an error on 4xx, and we can't use `apiFetch` since it adds a REST API prefix.

Reply:
I've added some handling for status codes, and rest error messages.